packages/typescript-estree/src/create-program/createProjectService.ts

```diff
@@ -0,0 +1,55 @@
+import * as tsserver from 'typescript/lib/tsserverlibrary';
```

Comment:
Is there any way that you can use a lazy import here, e.g. move this into a require in `createProjectService`? This will end up increasing the startup time of the plugin by another 140ms or so just from loading TS twice.

Reply:
Love it! Applying now, then will re-measure performance.